src/components/timeline/TrackVisualProgress.js

```diff
@@ -18,7 +18,7 @@ import type { ProgressGraphData, Milliseconds } from 'firefox-profiler/types';
 
 import type { ConnectedProps } from 'firefox-profiler/utils/connect';
 
-import './TrackVisualProgress.css';
+import 'firefox-profiler/components/timeline/TrackVisualProgress.css';
```

Comment:
Please revert this line, it starts with `./` and it's not related to `src/actions`.